skecon/utils.py

```diff
@@ -0,0 +1,47 @@
+import copy
+
+import torch
+
+from skecon.config import default_device, default_dtype
+
+@torch.no_grad()
+def _preprocess_data(data, dtype=None, device=None, shape=None,

```

Comment:
this is a horrendous name. will chage it to something like `astensor` or `asanytensor`